app/src/main/java/com/framgia/soundclound/screen/detailalbum/DetailAlbumModelView.java

```diff
@@ -1,18 +1,26 @@
 package com.framgia.soundclound.screen.detailalbum;
 
+import android.app.Activity;
 import android.app.AlertDialog;
 import android.content.Context;
 import android.content.DialogInterface;
+import android.content.Intent;
 import android.databinding.BaseObservable;
 import android.databinding.Bindable;
+import android.view.MenuItem;
```

Comment:
ERROR: Unused import: android.view.MenuItem. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports